volatility3/framework/plugins/windows/direct_system_calls.py

```diff
@@ -0,0 +1,455 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+import capstone
```

Comment:
Can we add import error protection around this (try except ImportError), so if they don't have it, it'll raise a nice error telling them they need to get it?  Also, do we want to add it to one of the requirements.txt files?

Reply:
done. We should add to requirements.txt for parity release given how many plugins need it.